File: Android/TheNPDev_HarshSingh_2125CSIT1050_2/GettingStarted/Print all primes till N.java
import java.util.*;

public class Main{
    public static void main(String[] args) {
        int div;
        //  write your code here

        Scanner sc = new Scanner(System.in);

        int low = sc.nextInt();
        int high = sc.nextInt();
        for(int num = low ; num <= high; num++)
        {
            for ( div = 2 ; div*div<= num; div++)
            {
                if(num%div == 0)
                {
                    break;
                }


            }
            if (div * div > num){
                System.out.println(num);
            }
        }
    }
}

File: Android/khushi1502_KhushiSachdev_2125cse1077_2/GettingStarted/PrintAllPrimesTillN.cpp
#include <iostream>
using namespace std;
int main() {
  int low, high;
  cin>>low>>high;

  for (int n = low; n <= high; n++) {
    int count = 0;
    for (int div = 2; div * div <= n; div++) {
      if (n % div == 0) {
        count++;
        break;
      }
    }

    if (count == 0) {
      cout<<n<<endl;
    }
  }

}